app/assets/javascripts/selectors/columns.js

```diff
@@ -0,0 +1,28 @@
+import { createSelector } from 'reselect';
+import { orderByState, groupStoriesInSprints } from "./backlog";
+import { mountPastIterations } from './done';
+import actionTypes from '../actions/actionTypes';
+import * as Column from "../models/beta/column";
+
+const getStories = (state) => state.stories;
+const getColumn = (state) => state.column;
+const getProject = (state) => state.project;
+const getPastIterations = (state) => state.pastIterations;
+
+export const getColumns = createSelector(
+  [getColumn, getStories, getProject, getPastIterations],
+  (column, stories, project, pastIterations) => {
+    switch (column) {
+      case actionTypes.COLUMN_CHILLY_BIN:
```

Comment:
Are these actual action types? Or just constants with the ids of the columns? I think we could move these constants out of `actionTypes.js`, what do you think?